examples/Demo/Shared/wwwroot/css/site.css

```
@@ -76,7 +75,8 @@ nav.sitenav {
     height: calc(100dvh - 90px);
     width: 330px;
     overflow-y: auto;
-
+    padding-left: 1.5rem;
+    padding-right: 1.2rem;
```

combine this with the `padding` at line 74 and make left and right the same 
```suggestion
    padding: 10px 1.5rem;
```

---

Also change the `width` to be in rem as well (and a bit smaller)
    width: 20rem;